packages/react/src/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,104 @@
+import type { Meta, StoryFn } from '@storybook/react';
+
+import {
+  EnvelopeClosedFillIcon,
+  HeartFillIcon,
+  PencilIcon,
+  VideoFillIcon,
+} from '@navikt/aksel-icons';
+import { Tabs } from '../Tabs';
+import { Badge } from './Badge';
+
+type Story = StoryFn<typeof Badge>;
+
+const meta: Meta<typeof Badge> = {
+  title: 'Komponenter/Badge',
+  component: Badge,
+  argTypes: {
+    size: {
+      options: ['sm', 'md', 'lg'],
+      control: { type: 'radio' },
+    },
+    color: {
+      options: ['accent', 'info', 'success', 'warning', 'danger', 'neutral'],
+      control: { type: 'select' },
+    },
+    placement: {
+      options: ['top-right', 'top-left', 'bottom-right', 'bottom-left'],
+      control: { type: 'radio' },
+    },
+  },
```

Comment:
Was this because of react-docgen problems? We stopped using `argTypes` as it leads to extra maintenance and prone for errors (as we manually need to update it manually). Description is also missing because of this.

Reply:
yes, something seems to have broken after we updated further.
See Button for example:
![image](https://github.com/user-attachments/assets/cb5ef8d4-289c-44a4-bdbc-5cfe2e5682cd)

Reply:
its probably because we disabled `react-docgen-typescript` locally.

Reply:
This screenshot is from https://next.storybook.designsystemet.no/?path=/docs/komponenter-button--docs